### DISCUSSION TITLE
how to check docmost version

### DISCUSSION THREAD

**Discussion by coodiecool** [docmost/docmost#345] [Help / Questions] [answered]
hi sir,
how to check the current version number?
I upgraded from 0.2.4 to 0.3.1, but I can't find the version number.
Best regards

**Comment by saventang** [comment]
open the Workspace settings

**Comment by coodiecool** [comment]
still can't find the version number
![docmost](https://github.com/user-attachments/assets/60408b23-e81e-4cb7-91de-a3ebf578e803)

**Reply by Philipinho** [reply]
Please post the server logs.

**Comment by Dunkon** [comment]
@coodiecool check docmost container logs and find follow lines, 0.3.1 is a version of app.
```bash
$ docker compose logs -f docmost
docmost-1  | 
docmost-1  | > docmost@0.3.1 start /app
docmost-1  | > pnpm --filter ./apps/server run start:prod
```
Or this for example
```bash
$ docker compose logs docmost | grep "docmost@"
docmost-1  | > docmost@0.3.1 start /app
```

**Reply by coodiecool** [reply]
hi sir,
The result of this method is the version number used during installation, not the upgraded version.

![docmost-1](https://github.com/user-attachments/assets/c49622e4-19d7-4cf4-a44f-8a1a7cc231cf)
![docmost-2](https://github.com/user-attachments/assets/a54e3b8f-f66b-4a3b-aa2d-d0d89c065725)

**Reply by Philipinho** [reply]
How did you upgrade the software?

**Comment by coodiecool** [comment, marked as answer]
I found out it was my problem,
I changed the image source of docker-compose.yml to 'docmost/docmost:latest' and ran 'docker compose up --force-recreate --build docmost -d' and the updated version number will be displayed.
![docmost-3](https://github.com/user-attachments/assets/91e31e5a-0292-48dc-89cb-1ef58848aea5)
